### DISCUSSION TITLE
Dashboard  jobs empty

### DISCUSSION THREAD

**Discussion by kirley** [bensheldon/good_job#1151] [Q&A] [answered]
My dashboard Jobs page is completely empty.  I have `config.good_job.preserve_job_records = true` but I don't see any history.  Jobs are running successfully.   I am running in a development environment.  I don't see anything in the documentation but i'm wondering if there is something that might be preventing the job history from showing?

**Comment by bensheldon** [comment, marked as answer]
That's strange! If you open up `bin/rails console` Can you try running:

- `GoodJob::Job.all.count` are there records in the database?
- If there aren't, run `YourJob.queue_adapter` and doublecheck that it is a GoodJob adapter

**Reply by kirley** [reply]
Thanks for your quick reply.  This helped me track down my problem.   I had `config.active_job.queue_adapter = :good_job` in application.rb but it was being overrwitten by 'config.active_job.queue_adapter = :async` in the development.rb.   So i commented out the line in development.rb and now Good_Job is working as expected.

**Comment by Pauloparakleto** [comment]
@bensheldon it helped me fix this in [my production enviroment](https://github.com/Purple-Stock/open-erp/pull/390). @kirley thanks to be the first asking about it.
